lib/apis/fetch.js

```diff
@@ -1,17 +1,27 @@
-import _ from 'lodash';
+import { defaults } from 'lodash';
 import request from 'request';
 import config from '../../config';
 
 export default (url, options = {}) => {
   return new Promise((resolve, reject) => {
-    const opts = _.defaults(options, { method: 'GET', timeout: config.REQUEST_TIMEOUT_MS });
+    const opts = defaults(options, {
+      method: 'GET',
+      timeout: config.REQUEST_TIMEOUT_MS,
+    });
+
     request(url, opts, (err, response) => {
-      if (!!err || response.statusCode !== 200) return reject(err || new Error(response.body));
+      if (!!err || response.statusCode !== 200) {
+        return reject(err || new Error(response.body));
+      }
 
       try {
         const shouldParse = typeof response.body === 'string';
         const parsed = shouldParse ? JSON.parse(response.body) : response.body;
-        resolve(parsed);
+
+        resolve({
```

Comment:
Weird—I am noticing that in most cases but in that case it's { body, headers}...

![image](https://cloud.githubusercontent.com/assets/555859/14793699/8684ed12-0aee-11e6-8442-24ec8505423e.png)